### ISSUE TITLE
San sterling méret

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#31]
```
Game version:0.93

Déli irányban legalább egy "kórháznyival" nagyobb az építési felület. 
Eredetiben délre szoktam lerakni mindhárom gyárat+ a katonai űrkikötőt, 
itt nem fér el egyik sem.
```

Original issue reported on code.google.com by `Jozsef.T...@gmail.com` on 16 Aug 2011 at 7:57

**Comment by GoogleCodeExporter** [comment]
```
Küldjél légyszi képet az eredetiről. Valószínűleg túlszabtam azt a 
bolygótípust.
```

Original comment by `akarn...@gmail.com` on 16 Aug 2011 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
Nem a legjobb kép, de azért a radaron látszik, hogy jónéhány nagy 
épületet le tudtam rakni a jéghegy mögé délen. A jobb szélén pl ott 
figyel egy stadion!
```

Original comment by `Jozsef.T...@gmail.com` on 16 Aug 2011 at 8:22

Attachments:
- [sansterling.jpg](https://storage.googleapis.com/google-code-attachments/open-ig/issue-31/comment-2/sansterling.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Kösz. Kb 5 sorral kevesebb volt a Frozen-2 típusú bolygón.
```

Original comment by `akarn...@gmail.com` on 16 Aug 2011 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Fix 0.93.100
```

Original comment by `akarn...@gmail.com` on 17 Aug 2011 at 11:12
- Changed state: **Fixed**
